eng/targets/Npm.Common.targets

```diff
@@ -31,8 +36,14 @@
              BuildInParallel="true" />
   </Target>
 
-  <Target Name="Build" DependsOnTargets="PrepareForBuild;ResolveProjectReferences">
-    <Yarn Command="run build" Condition="'$(IsBuildable)' != 'false'" StandardOutputImportance="High" StandardErrorImportance="High" />
+  <Target Name="Build" DependsOnTargets="PrepareForBuild;ResolveProjectReferences;_Build" />
+
+  <Target Name="_Build"
+    Condition="'$(IsBuildable)' != 'false'"
+    Inputs="@(TSFiles)"
+    Outputs="$(BaseIntermediateOutputPath)\build-sentinel" >
```

Comment:
What's the build-sentinel do?

Reply:
Its for incremental builds. Msbuild will look for it when building a project and compare the timestamp vs. the input files `"@(TSFiles)"` to determine if it should build the project.